### ISSUE TITLE
Support environment variable use within .env file.

### ISSUE THREAD

**Issue by jpickwell** [DarthSim/overmind#27]
```bash
# .env
NODE_ENV=development
RAILS_ENV=${NODE_ENV}
```

The above doesn't work correctly. `RAILS_ENV` gets set to the literal value `${NODE_ENV}` instead of the correct value of `development`.

OS: macOS 10.12.6 w/ Homebrew
tmux: 2.6
Overmind: 1.1.1

**Comment by prikha** [comment]
Well I would widen the scope - ENV vars are not propagated at all, even when you have 

```
> echo $PORT
3003

> cat Procfile
web: bin/rails s -p $
```

**Comment by jpickwell** [comment]
I agree that global environment should be propagated to the tmux environment.
